synapseclient/core/utils.py

```diff
@@ -206,13 +222,15 @@ def concrete_type_of(obj: collections.abc.Mapping):
     return concrete_type
 
 
-def is_in_path(id, path):
+def is_in_path(id: str, path: collections.abc.Mapping) -> bool:
     """Determines whether id is in the path as returned from /entity/{id}/path
 
-    :param id:      synapse id string
-    :param path:    object as returned from '/entity/{id}/path'
+    Arguments:
+    id: synapse id string
```

Comment:
Tab these in one otherwise it'll break formatting:

![image](https://github.com/Sage-Bionetworks/synapsePythonClient/assets/17128019/244156af-83bd-4fda-90d2-732a91f9f798)